validation/activity-stream/pocket-button.1.door_hanger_click.pass.json

```diff
@@ -0,0 +1,14 @@
+{
+  "locale": "en-US",
+  "impression_id": "{94642acb-4996-034b-916c-147da723cc41}",
+  "version": "72.0a1",
+  "release_channel": "nightly",
+  "events": [
+    {
+      "action": "click",
+      "source": "learn_more"
```

Comment:
Possible `source` values for links on the door-hanger menu are:
-`learn_more`
-`log_in`
-`sign_up_1`
-`sign_up_2`

I am not making this an enum, as we may introduce new values in the future.

Reply:
Note that string fields and enums are both represented as STRING in BQ, so you're free to update this detail in the schema going forward. If you want to leave this loose here, that's fine. But it's also an option to make it an enum with known existing values, and make future PRs here to add new values before you start sending pings with those values. If this is likely to evolve frequently, then the enum is likely to cause more pain than benefit.

Reply:
I'd like to leave it as an open string for now, thanks for the detail!